b2/console_tool.py

```diff
@@ -344,6 +344,25 @@ def run(self, args):
         return 0
 
 
+class GetFileInfoAndDeleteFileVersion(Command):
```

Comment:
Maybe `delete_file_id` would be more concise, and work better with file completion.